scalajvm/app/views/main.scala.html

```diff
@@ -24,6 +24,6 @@
             <li class="menu-service"><a href="@routes.SuggestedQuotes.newQuote()">Добавить цитату</a></li>
 	    </ul>
         <div class="content center-content">@content</div>
-        <div class="footer center-content"><span class="highlight"><a href="https://github.com/codingteam/loglist">codingteam</a> ©</span> 2015</div>
+        <div class="footer center-content"><span class="highlight"><a href="https://github.com/codingteam/loglist">codingteam</a> ©</span> 2016</div>
```

Comment:
I am not sure we need that. I've found [the following article](http://www.contentious.com/2007/01/07/copyright-notice-is-the-year-really-necessary/) and decided not to use range here (especially because we're [already not using ranges since 2015](https://github.com/codingteam/loglist/commit/141d2dc9693fa8cb3c3e5575a7e345d50c6dd03f)).